docs/Catalog/Preview.md

```diff
@@ -29,14 +29,25 @@ The following file formats are supported:
 * .ent
 * .pdb
 
+## Image previews
+
+The Quilt catalog uses the [AICSImageIO
+library](https://github.com/AllenCellModeling/aicsimageio) to
+automatically generate thumbnail previews of multiple
+microscopy image file formats.
```

Comment:
The catalog uses Lambda and the Lambda uses aicsimageio (at a specific lower version that is not top of tree in their GH, so linking to their GitHub is not useful. Instead, link to or just refer to the open source thumbnail lambda code.